sqlflow/__main__.py

```diff
@@ -0,0 +1,24 @@
+import argparse
+
+from sqlflow.client import Client
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--url", type=str, help="server url", action="store", required=True)
+
+def main():
+    args = parser.parse_args()
+
+    client = Client(server_url=args.url)
+
+    print('Standard SQL, prints dataframes')
+    response = client.execute('select * from t')
+    for dataframe in response:
+        print(dataframe) # {'y': [4, 5, 6], 'x': [1, 2, 3]}
+
+    print('Extended SQL, prints epoch = ..., loss = ...')
+    # TODO(tonyyang-svail): avoid for loop
+    for log in client.execute('SELECT * TRAIN .'):
+        pass
+
+if __name__ == '__main__':
```

Comment:
It may replace this with entry point

Reply:
also accept sql(s) as argument, so user can run it in terminal:

sqlflow sql(s) [--server_url=xxx]